begrun.c

```diff
@@ -196,3 +202,105 @@ void close_outputfiles(void)
 }
 
 
+/*! Computes conversion factors between internal code units and the
+ *  cgs-system.
+ */
+static void
+set_units(void)
+{
```

Comment:
I ask because I wanted to do this for the neutrinos the other day, but could not be bothered to split out the header reading :)